cli/copr_cli/main.py

```diff
@@ -325,7 +325,15 @@ def action_build(self, args):
         # Before we start uploading potentially large source RPM file, make sure
         # that the user has valid credentials and can build in the project.
         self.client.build_proxy.check_before_build(
-            username, projectname, project_dirname, buildopts)
+            ownername=username,
+            projectname=projectname,
+            # documentation says user can create directory with copr build, but
+            # /build/check-before-build endpoint checks for presence of this
+            # dirname even before creating it in this check. Passing only projectname
+            # should be sufficient for the comment above.
+            project_dirname=projectname,
```

Comment:
Have you tried to call this with `project_dirname=None`?

Reply:
Yes and it failed with `Error: The browser (or proxy) sent a request that this server could not understand`. If `project_dirname` is not passed to cli and only projectname is passed in either `ownername/projectname` or only `projectname` format then the project_dirname is not `None` but `projectname`.

Reply:
This is a frontend bug, isn't it?

Reply:
Note this:
https://github.com/fedora-copr/copr/blob/3b68ad13a4f44229ecb02ef5d9677278a7a69c2e/frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_builds.py#L230-L232